src/open_inwoner/components/templates/components/Step/StepIndicator.html

```diff
@@ -4,7 +4,7 @@
     <aside class="step-indicator" aria-label="{% trans "Stappen indicator" %}">
         <ol class="step-indicator__list">
             {% for step in steps %}
-                <li class="step-indicator__list-item{% if step.is_current %} step-indicator__list-item--active{% elif step.is_completed %} step-indicator__list-item--completed{% endif %} P" data-step={{ step.step }} aria-checked="{{ step.is_completed|yesno:"true,false" }}">
+                <li class="step-indicator__list-item{% if step.is_current %} step-indicator__list-item--active{% elif step.is_completed %} step-indicator__list-item--completed{% endif %} P" data-step={{ step.step }} {% if step.is_current %}aria-current="step"{% endif %} aria-label="{{ step.is_completed|yesno:_("afgerond,huidige stap") }}">
```

Comment:
Not your change, but what is the "P" in the `li` class? Is that an actual css class we're using?